src/App.tsx

```diff
@@ -206,6 +206,17 @@ function App() {
                 </small>
               </li>
 
+              <li>
+                For convenience, you can store trust anchors and safebags in QR code and scan them for quick fill-in. On
+                Linux-based OS's, the following command converts text to QR code (using trust anchor as example):
+                <div style={{ 'padding-left': '15px' }}>
+                  <code>ndnsec cert-dump -i /my-workspace | qrencode -o - | feh -</code> <br />
+                </div>
+                <small>
+                  To install the necessary packages, use the command <code>apt install</code>
```

Comment:
How about putting it as the first command, saying `apt install qrencode feh`, then run the second line to display